python/pyarrow/tests/test_compute.py

```diff
@@ -1347,6 +1351,32 @@ def test_strptime():
     assert got == expected
 
 
+# TODO: We should test on windows once ARROW-13168 is resolved.
+@pytest.mark.pandas
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Timezone database is not available on Windows yet")
+def test_strftime():
```

Comment:
Can you add some examples with `%z`/`%Z` on both naive/local timestamps and UTC timestamps.  It looks like python emits an empty string for `%z`/`%Z` if the timestamp is naive but I don't know what the vendored tz lib will do.

Reply:
date.h will fail on naive timestamp:
> ***%z*** | The offset from UTC in the ISO 8601 format. For example -0430 refers to 4 hours 30 minutes behind UTC. If the offset is zero, +0000 is used. The modified commands %Ez and %Oz insert a : between the hours and minutes: -04:30. If the offset information is not available, failbit will be set.

I'll see if I can make it pandas-like.

Reply:
Done.